ipamd/datastore/data_store.go

```diff
@@ -41,19 +41,23 @@ var ErrUnknownPodIP = errors.New("datastore: pod using unknown ip address")
 
 // ENIIPPool contains ENI/IP Pool information
 type ENIIPPool struct {
-	createTime            time.Time
-	lastUnassignedTime    time.Time
-	isPrimary             bool
-	id                    string
-	deviceNumber          int
-	assignedIPv4Addresses int
-	ipv4Addresses         map[string]*AddressInfo
+	createTime         time.Time
+	lastUnAssignedTime time.Time
+	// IsPrimary indicates whether eni is a primary eni
+	IsPrimary bool
+	id        string
+	// DeviceNumber is the device number of eni
+	DeviceNumber int
+	// AssignedIPv4Addresses is the number of ip addesses already been assigned
+	AssignedIPv4Addresses int
+	// IPv4Addresses shows whether each address is assigned
+	IPv4Addresses map[string]*AddressInfo
```

Comment:
I presume these name changes are to export the struct members for visibility in the introspection package; why exclude the time fields then?  Also the nice formatting alignment got messed up.  Oh, and please consistently capitalize initialisms ENI,IP,etc in the comments.